src/development/tools/sdk/release-notes/supported-platforms.md

```diff
@@ -78,17 +78,15 @@ minimal.
 |Windows |Windows 7      |
 |Linux   |Debian & below |
 
-We have dropped iOS8 support. For more information,
-see [go/rfc-ios8-deprecation] for details.
-
-[go/rfc-ios8-deprecation]: {{site.url}}/go/rfc-ios8-deprecation
-
 ### Unsupported platforms
 
 |Platform|Version               |
 |--------|----------------------|
 |Android |Android SDK 18 & below|
-|iOS     |iOS 8 & below         |
+|iOS     |[iOS 8] & below and [`arm7v` 32-bit iOS]|
```

Comment:
Hopefully I did this link-in-table correctly...

Reply:
I think it looks right. The link checker should complain if it's wrong.

PTAL @khanhnwin =)